docs/src/main/resources/microsite/data/menu.yml

```diff
@@ -109,13 +109,45 @@ options:
       - title: Hammock
         url: docs/integrations/hammock
 
-  - title: Libraries
+  - title: Microservices
     url: docs/rpc
 
     nested_options:
 
-      - title: Microservices
-        url: docs/rpc
+      - title: Quickstart
+        url: docs/rpc/quickstart
+
+      - title: Core concepts
+        url: docs/rpc/core-concepts
+
+      - title: Implementations
```

Comment:
Related to https://github.com/frees-io/freestyle-rpc/pull/190#discussion_r172783943